frontend/src/metabase/dashboard/actions/auto-wire-parameters/toasts.ts

```diff
@@ -7,64 +7,168 @@ import {
   setMultipleDashCardAttributes,
 } from "metabase/dashboard/actions";
 import { addUndo, dismissUndo } from "metabase/redux/undo";
-import type { QuestionDashboardCard, DashCardId } from "metabase-types/api";
+import type {
+  QuestionDashboardCard,
+  DashCardId,
+  DashboardParameterMapping,
+  Parameter,
+} from "metabase-types/api";
 import type { Dispatch } from "metabase-types/store";
 
 export const AUTO_WIRE_TOAST_ID = _.uniqueId();
+const AUTO_WIRE_UNDO_TOAST_ID = _.uniqueId();
 
 export const showAutoWireParametersToast =
   ({
     dashcardAttributes,
+    originalDashcardAttributes,
+    fieldName,
+    multipleTabs,
   }: {
     dashcardAttributes: SetMultipleDashCardAttributesOpts;
+    originalDashcardAttributes: SetMultipleDashCardAttributesOpts;
+    fieldName: string;
+    multipleTabs: boolean;
```

Comment:
nit: `hasMultipleTabs`? It was somewhat unclear it's a boolean

Reply:
[`56f6b6e` (#43191)](https://github.com/metabase/metabase/pull/43191/commits/56f6b6e9475b964e003ff814acc5968d446e2e42)